app/src/main/java/com/example/theperiodpurse/ui/setting/SettingScreenLayout.kt

```diff
@@ -0,0 +1,179 @@
+    onBackUpClicked: () -> Unit,
+    onDeleteClicked: () -> Unit,
+){
+   Column(modifier = modifier
+       .fillMaxSize()
+       .verticalScroll(rememberScrollState())) {
+       Text(
+           text = stringResource(R.string.tracking_preferences),
+           modifier = modifier.padding(
+               top = 50.dp,
+               start = 10.dp),
+           color = Color.Gray
+       )
+
+       TrackingPreferencesRow()
+       Text(
+           text = stringResource(R.string.notifications_heading),
+           modifier = modifier.padding(
+               top = 50.dp,
+               start = 10.dp),
+           color = Color.Gray
+       )
+       Row(modifier = modifier.padding(20.dp)) {
+           var checked by remember { mutableStateOf(false) }
+           Column (modifier = Modifier) {
+               Text(text = stringResource(
+                   R.string.remind_me_to_log_symptoms),
+                   fontWeight = FontWeight.Bold)
+               Spacer(modifier = modifier.padding(3.dp))
+               Text(text = stringResource(R.string.sample_notification))
+           }
+           Switch(
+               checked = checked,
+               onCheckedChange = { checked = !checked},
+               modifier = modifier
+                   .fillMaxWidth()
+                   .wrapContentWidth(Alignment.End),
+               colors = SwitchDefaults.colors(
+                   uncheckedThumbColor = Color.DarkGray
+               )
+           )
+       }
+      TabOption(
+          stringResource(id = R.string.customize_notifications),
+          onTabClicked = onNotificationClicked
+      )
+       Text(
+           text = stringResource(R.string.account_settings_heading),
+           modifier = modifier.padding(
+               top = 50.dp,
+               start = 10.dp),
+           color = Color.Gray
+       )
+       TabOption(
+           text = stringResource(R.string.back_up_account),
+           onTabClicked = onBackUpClicked
+       )
+       Spacer(modifier = Modifier.padding(10.dp))
+       TabOption(
+           text = stringResource(id = R.string.delete_account),
+           onTabClicked = onDeleteClicked
+       )
+       Spacer(modifier = Modifier.padding(10.dp))
+       Text(text = "The Period Purse. All rights reserved.")
+   }
+}
+
+@Composable
+fun TrackingPreferencesRow(modifier: Modifier = Modifier){
+    Row(modifier = modifier
+        .fillMaxWidth()
+        .padding(30.dp),
+        verticalAlignment = Alignment.CenterVertically,
+        horizontalArrangement = Arrangement.Center) {
+        TrackingOptionButton(
+            modifier = modifier,
+            label = stringResource(R.string.mood),
+            icon = painterResource(id = R.drawable.sentiment_neutral_black_24dp)
+        )
+        TrackingOptionButton(
+            modifier = modifier,
+            label = stringResource(R.string.exercise),
+            icon = painterResource(id = R.drawable.self_improvement_black_24dp)
```

Comment:
Looks like we will be using these label and icon pairs on a few different pages (`R.string.mood matches` with `R.drawable.sentiment_neutral_black_24dp`), such as the onboarding pages and the calendar page.

We can look into modifying the `symptoms` object in `DataSource` to include this information for reusability. Perhaps we can create an issue for this sort of refactoring/enhancement?

Reply:
This is the only change that I didn't make yet. It seems like this should be for a different PR, but lmk if theres anything to change now as well